src/components/select/index.js

```diff
@@ -33,21 +34,24 @@ export default class Select extends React.Component {
   }
   render() {
     return (
-      <div className="select-wrapper">
+      <div
+        className={ classnames('select-wrapper', this.props.className, {
+          'select-wrapper--empty': !this.state.label && !this.props.label
```

Comment:
`!this.props.label` was also necessary, otherwise the select with default value was grayed out too.